jmxtrans-core/src/main/java/com/googlecode/jmxtrans/JmxTransformer.java

```diff
@@ -550,8 +579,7 @@ private void scheduleReload() {
 			@Override
 			public void run() {
 				try {
-					deleteAllJobs();
-					startupSystem();
+					restartSystem();
 				} catch(Exception e) {
 					throw new RuntimeException(e);
```

Comment:
Does it make sense, to move the try-catch in the restartSystem method and remove the throws Exception from signature

Reply:
This code is actually changed in a subsequent PR (#714); throwing a RuntimeException isn't the right thing to do here anyway.